sdk/attestation/attestation/src/attestationAdministrationClient.ts

```diff
@@ -237,5 +260,26 @@ export class AttestationAdministrationClient {
     }
   }
 
+  private get _signingKeys(): Promise<AttestationSigner[]> {
+    if (this._signers !== undefined) {
+      return Promise.resolve(this._signers);
+    }
+    const signingCertificates = this._client.signingCertificates.get();
```

Comment:
NIT: I think this code after this could be simplified by awaiting this call so we do not have to use callbacks.

Do we have the guarantee that these certs are not going to change by another client or direct interactions with the service?

Reply:
If the service signing key was rolled over between the first time this was set, and when the call is made, they can change. That happens about once every 5 years.

And I tried to use await, but TypeScript really didn't like the idea of async properties, so I had to do it the hard way :(.